scripts/utils/cliContextUtils.ts

```diff
@@ -0,0 +1,62 @@
+import type { RequestContext } from '@lokalise/fastify-extras'
+import { generateMonotonicUuid } from '@lokalise/id-utils'
+import type { CommonLogger } from '@lokalise/node-core'
+import type z from 'zod'
+import { type AppInstance, getApp } from '../../src/app.js'
+
+export const getArgs = () => {
+  return process.argv.reduce((args: Record<string, unknown>, arg) => {
```

Comment:
is it possible to simplify this code by using https://nodejs.org/api/util.html#utilparseargsconfig?

Reply:
Good catch! Added.